src/lib/components/ai-model-form.svelte

```diff
@@ -33,7 +35,17 @@
 	const aiFormId: string = crypto.randomUUID();
 </script>
 
-<h1 class="text-sm font-bold uppercase text-muted-foreground">Configuración AI</h1>
+<h1 class="flex items-center gap-2 text-sm font-bold uppercase text-muted-foreground">
+	Configuración AI 
+	<span>
+		<button
+			onclick={() => {
+				model.reset();
+				isDisabled = false;
+			}}><svelte:component this={Icons['pencil']} class="h-4 w-4" />
+		</button>
```

Comment:
Refactor this to:

‘<Icons.pencil class=“h-4 w-4” />’

Reply:
Done!